lambda/cfn.yaml

```diff
@@ -92,3 +92,39 @@ Resources:
       Timeout: 300
     DependsOn:
       - PriceMigrationEngineLambdaRole
+  PriceMigrationEngineSalesforcePriceCreationLambda:
+    Type: AWS::Lambda::Function
+    Properties:
+      Description: Lambda used to create Price_Rise__c objects in salesforce realting to the price rise.
+      FunctionName:
+        !Sub price-migration-engine-salesforce-price-rise-lambda-${Stage}
+      Code:
+        S3Bucket: membership-dist
+        S3Key: !Sub membership/${Stage}/price-migration-engine-lambda/price-migration-engine-lambda.jar
+      Handler: pricemigrationengine.handlers.SalesforcePriceRiseCreationHandler::handleRequest
+      Environment:
+        Variables:
+          stage: !Ref Stage
+          zuoraApiHost:
+            !Sub
+              - '{{resolve:secretsmanager:price-migration-engine-lambda-${Stage}:SecretString:zuoraApiHost::${SecretsVersion}}}'
+              - SecretsVersion: !FindInMap [StageMap, !Ref Stage, SecretsVersion]
+          zuoraClientId:
+            !Sub
+              - '{{resolve:secretsmanager:price-migration-engine-lambda-${Stage}:SecretString:zuoraClientId::${SecretsVersion}}}'
+              - SecretsVersion: !FindInMap [StageMap, !Ref Stage, SecretsVersion]
+          zuoraClientSecret:
+            !Sub
+              - '{{resolve:secretsmanager:price-migration-engine-lambda-${Stage}:SecretString:zuoraClientSecret::${SecretsVersion}}}'
+              - SecretsVersion: !FindInMap [StageMap, !Ref Stage, SecretsVersion]
+          earliestStartDate: 2020-05-15
```

Comment:
I don't think any of this will be needed

Reply:
I will have to create separate Config classes and config services for each of the lambdas if we dont want to use the same config for each app.

Which is probably a good thing. Ill do it

Reply:
It turns out to be a bit of a headache to have different config for each handler.

This is because of the shared services eg the DynamoDbLive class is dependent on Configuration, if I create a new config class eg SalesforcePriceRiseConfiguration then I cant wire the DynamoDbLive service into the handler that uses that configuration.

I think the best way would be to create completely separate config case classes/loader services for each of the service ie Zoura/Salesforce/DynamoDB and one each for the handlers with any of the remaining config items that are specific to that handler in it eg earliestStartDate

Reply:
Ok that is done, there is quite a bit of change in this PR now, but the config is much cleaner.

Reply:
Great work!